GoogleDataTransport/GDTCORTests/Unit/Helpers/GDTCOREventGenerator.m

```diff
@@ -26,14 +26,14 @@ @implementation GDTCOREventGenerator
 
 + (NSMutableSet<GDTCOREvent *> *)generate3Events {
   static NSUInteger counter = 0;
-  NSString *filePath = [NSString stringWithFormat:@"test-%ld.txt", (unsigned long)counter];
   int howManyToGenerate = 3;
   NSMutableSet<GDTCOREvent *> *set = [[NSMutableSet alloc] initWithCapacity:howManyToGenerate];
   for (int i = 0; i < howManyToGenerate; i++) {
     GDTCOREvent *event = [[GDTCOREvent alloc] initWithMappingID:@"1337" target:50];
     event.clockSnapshot = [GDTCORClock snapshot];
     event.qosTier = GDTCOREventQosDefault;
     event.dataObject = [[GDTCORDataObjectTesterSimple alloc] initWithString:@"testing!"];
+    NSString *filePath = [NSString stringWithFormat:@"test-%ld.txt", (unsigned long)counter];
```

Comment:
Why not use `i` instead of `counter`?

Reply:
Because using i could result in creating duplicate event fileURLs across multiple calls to this method